scripts/sql-ing-lookup.py

```diff
@@ -0,0 +1,80 @@
+#%%
+# Define function to query details for ingredients
+def scmd_ingredient_query(ingredient_list, str_to_lower = True, wildcards = "both"):
+    '''
+    Define SQL query searching for all ingredients specified in 'ingredient_list'.
+    The returned data includes the following columns:
+    
+    Arguments:
+    ingredient_list {list of strings}: for example ["drug1", "drug2", "drug3"]
+    wildcards {string}: "prefix", "suffix", "both", "none"
```

Comment:
You can also list Outputs here

Reply:
There are several (different) style guides for docstrings, but I think we more-or-less follow the [Google Python Style Guide](https://google.github.io/styleguide/pyguide.html), which has a section on [Comments and Docstrings](https://google.github.io/styleguide/pyguide.html#s3.8-comments-and-docstrings).

In this context, I'd argue that the formatting doesn't matter nearly as much as the content. Note the types and the permitted values of the arguments, where these aren't clear (e.g. `ingredient_list` is a `list`, isn't it?). Similarly, note the type of the return value.